src/report-embedded-add/report-embedded-add.html

```diff
@@ -0,0 +1,69 @@
+<h2>{{'adminReportAdd.title' | message}}</h2>
```

Comment:
As a small optimalization improvement you can make angularjs load this translation only once. Normally it will reevaluate it in every digest cycle. The same applies to other translations (with exception of dynamic ones which use report.name)

`{{'adminReportAdd.title' | message}}`

https://docs.angularjs.org/guide/expression#one-time-binding